### DISCUSSION TITLE
Power and Screen On, No Display

### DISCUSSION THREAD

**Discussion by Publ1cAnem0ne** [sanni/cartreader#911] [Need help]
Hey there! I'm hoping for some assistance in troubleshooting. 

I'm building the V5 hardware with VSELECT. All parts were purchased from Bonzo's Retry Shop back in November. 

I'm at [Step 6](https://github.com/sanni/cartreader/wiki/How-to-build#6-the-first-test) in the How to Build guide, which is performing the first test. I appear to be getting power, the screen lights up, but there is nothing on the screen. 

I tried getting the device output in Serial Monitor, but the Arduino software isn't seeing the device on any COM ports. If I go into device manager, it keeps flashing every couple of seconds like it's seeing a new device and then it disconnects. 

When I peek at the Mega Pro's LED, I notice it briefly shows a green LED, quickly turns red, then turns off. It continues to do this in a loop:
https://github.com/sanni/cartreader/assets/57876984/8af9cb18-1ae1-42d7-b7ee-ea5598fdf325

I've inspected all solder points on the main board, Mega Pro, and LCD module and there does not appear to be bridged connections. To verify, I tested continuity on all adjacent pins that were soldered with a multimeter, which I found none except for grounding pins. 

One thing that I did find is if I remove the Mega Pro from the main board and jump the connection where the fuse is removed in [Step 3](https://github.com/sanni/cartreader/wiki/How-to-build#3-arduino-compatible-mega-2560-pro-embed), then plug it into my computer, Device Manager sees it on COM3 with no issues and so does the Arduino app, though Serial Mode produces no output. 

Any suggestions before I start ordering replacement parts to try to component-isolate? 

Appreciate you all!

**Comment by breyell** [comment]
Hi, Bonzo here. Sorry to hear about the trouble. Any chance you use Discord and I could help you troubleshoot there? Would you mind posting some pictures of your work so far?

https://discord.gg/32aBcWRKRK

**Reply by Publ1cAnem0ne** [reply]
Absolutely! Taking this over to Discord now, but will chime back in here with what we find.

**Comment by Ancyker** [comment]
The 3.3V jumper on the LCD probably isn't closed.

Edit: Take note of the 2nd picture: https://github.com/sanni/cartreader/wiki/How-to-build#2-lcd-module

**Reply by Publ1cAnem0ne** [reply]
I do have it closed. Bonzo suggested I resolder it and I did. I verified that the left pad is getting continuity with right right, as well as the pin at the bottom of U1 and the right pin of C4. Still no luck. 

My next step is I'm waiting for another Megato arrive so that I can burn the boot loader, following the instructions here: https://github.com/sanni/cartreader/issues/619

**Reply by Ancyker** [reply]
Check while it's turned on if there is 3.3V getting to the OSCR because it looks like there isn't.

**Reply by Publ1cAnem0ne** [reply]
Got it. I'm measuring the voltage between the 5V and GND pins on the back of the Mega while everything is connected and getting between .9 and 3.4V (it keeps changing). Does this seem correct? Are there a different set of contacts that I should be measuring between?

**Reply by Ancyker** [reply]
Try testing 5V and 3.3V where the voltage select switch normally is.

**Comment by Krystman** [comment]
Just wanted to make sure - did you flash the Mega with the firmware with ENABLE_VSELECT on?

Otherwise, can you check the voltage on each of the the 3 pads where you would install the voltage select switch? From back to front we should see 5V, 3.3V and 3.3V

<img width="320" alt="Screenshot 2024-03-11 160130" src="https://github.com/sanni/cartreader/assets/10092987/be3420e7-bbac-4409-848f-06326807e64f">

**Reply by Publ1cAnem0ne** [reply]
I did verify that #define ENABLE_VSELECT was uncommented. Since I'm using VSELECT, wouldn't it cause damage if I jumped those pins to measure the voltage?

**Reply by Krystman** [reply]
I'm not quite sure what your concern is. Generally a Multimeter shouldn't cause any shortcuts. Also you wouldn't be jumping the pads. You would connect the red probe of your Multimeter to one of the pads and the black probe to ground somewhere. The 5V pad should have 5V coming from the USB port over the wire you soldered to the Mega. The lower 3.3 pad should tell us if the voltage regulator of the display is working correctly. The middle pad tells us what the VSELECT chip is doing. You probably measured that last time already when you were checking the 5V pin on the Mega. But it is good to check again.

**Reply by Publ1cAnem0ne** [reply]
Apologies for the late reply and thank you for clarifying.

There 5V pad is getting 5V consistently, middle pad is getting jumping between 3.5V and .8V, the bottom 3.3V pad is jumping between 3V and .9V.

**Comment by Publ1cAnem0ne** [comment]
I ordered a replacement Mega and screen. I tried using the new Mega to reload the boot loader on my original Mega, but had no luck. I moved on to uploading the Sketch in Arduino IDE to the new Mega, issue persisted. I jumped the two pads on the new screen, connected that, and now it's starting up, displays the menu on the screen, and allows me to read GameBoy cartridges. Just to isolate, if I have either the old Mega or old screen connected with a new Mega or new screen, the issue persists. It only works when I have both of the new components installed. I'm not sure why, but I'm happy that I have it working. 

![IMG_2625](https://github.com/sanni/cartreader/assets/57876984/18084fde-9b16-4730-a88d-a61e6042ae11)

**Reply by cmcm711** [reply]
I've had Megas in the past with power issues where they would either not run properly (or at all) at 3.3v, and some that even overheated so much once plugged in that they shut themselves down (no communication to PC, so wasn't even able to flash it). So, your issues aren't unheard of.
